NullPointerException in GppLinker if g++ is not installed

I've been experimenting with this plugin and encountered a NullPointerException when compiling my project.  I successfully compiled on OS X but my new CentOS 7 VM threw this NPE during the build:
```
[ERROR] Failed to execute goal com.github.maven-nar:nar-maven-plugin:3.2.0:nar-compile (default-nar-compile) on project webp-imageio: Execution default-nar-compile of goal com.github.maven-nar:nar-maven-plugin:3.2.0:nar-compile failed. NullPointerException -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal com.github.maven-nar:nar-maven-plugin:3.2.0:nar-compile (default-nar-compile) on project webp-imageio: Execution default-nar-compile of goal com.github.maven-nar:nar-maven-plugin:3.2.0:nar-compile failed.
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:224)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:153)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:145)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:116)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:80)
	at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:51)
	at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:120)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:347)
	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:154)
	at org.apache.maven.cli.MavenCli.execute(MavenCli.java:582)
	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:214)
	at org.apache.maven.cli.MavenCli.main(MavenCli.java:158)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:289)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)
	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)
	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)
Caused by: org.apache.maven.plugin.PluginExecutionException: Execution default-nar-compile of goal com.github.maven-nar:nar-maven-plugin:3.2.0:nar-compile failed.
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:143)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:208)
	... 19 more
Caused by: java.lang.NullPointerException
	at com.github.maven_nar.cpptasks.gcc.GppLinker.isClang(GppLinker.java:320)
	at com.github.maven_nar.cpptasks.gcc.GppLinker.addImpliedArgs(GppLinker.java:162)
	at com.github.maven_nar.cpptasks.compiler.CommandLineLinker.createConfiguration(CommandLineLinker.java:156)
	at com.github.maven_nar.cpptasks.compiler.AbstractLinker.createConfiguration(AbstractLinker.java:86)
	at com.github.maven_nar.cpptasks.ProcessorDef.createConfiguration(ProcessorDef.java:203)
	at com.github.maven_nar.cpptasks.CCTask.collectExplicitObjectFiles(CCTask.java:438)
	at com.github.maven_nar.cpptasks.CCTask.execute(CCTask.java:585)
	at com.github.maven_nar.NarCompileMojo.createLibrary(NarCompileMojo.java:431)
	at com.github.maven_nar.NarCompileMojo.narExecute(NarCompileMojo.java:105)
	at com.github.maven_nar.AbstractNarMojo.execute(AbstractNarMojo.java:302)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:132)
	... 20 more
```

I tracked the problem in GppLinker.isClang which is trying to invoke "g++ --version".  Since this is a new VM, I was still in the process of determining what needed to be installed.  In this case, I hadn't yet installed the gcc-c++ package to get g++.

The code should check for null and notify the user of a problem.

--- a/src/main/java/com/github/maven_nar/cpptasks/gcc/GppLinker.java
+++ b/src/main/java/com/github/maven_nar/cpptasks/gcc/GppLinker.java
@@ -330,6 +330,6 @@ public class GppLinker extends AbstractLdLinker {
         command, "--version"
     };
     final String[] cmdout = CaptureStreamHandler.execute(cmd).getStdout();
-    return cmdout.length > 0 && cmdout[0].contains("(clang-");
+    return cmdout != null && cmdout.length > 0 && cmdout[0].contains("(clang-");
   }
 }

